### DISCUSSION TITLE
3.89.10 (Android)

### DISCUSSION THREAD

**Discussion by charlag** [tutao/tutanota#3629] [Bugs]
# What's new
- Android: Provide option to lock saved credentials with PIN/biometrics #3588

# Milestone
https://github.com/tutao/tutanota/milestone/183?closed=1

# APK Checksum
SHA256: dc32889ddcd2e1b7587ab1359b555ea9216fae93b0d0a7d96b86e06517ed2519

<hr /><em>This discussion was created from the release <a href='https://github.com/tutao/tutanota/releases/tag/tutanota-android-release-3.89.10'>3.89.10 (Android)</a>.</em>

**Comment by lipengyuan1994** [comment]
Nice!
